### PR TITLE
Stream

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 35a85c511262414290bf7690230b7d2d012b6417
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**

 * Commit: 68ced10f296e257001f7ca95b4b4295209cbc64b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: d91f67b372e318f6a3bceecf8ae2f7338af73cc4
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 4f9ac84acdd10d3788956242224da4e26e39e442
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by joshuawilson** [comment]
Can you please rebase this, since we already have the revert in place.

File: src/app/profile/spaces/spaces.component.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export class SpacesComponent implements OnInit {
     private spaceService: SpaceService,
     private logger: Logger,
     private contexts: Contexts) {
-      this.contexts.current.subscribe(val => this.context = val);
+    this.contexts.current.subscribe(val => this.context = val);
```
Comment (joshuawilson) [inline src/app/profile/spaces/spaces.component.ts]: I'm not sure why your IDE changed this, but it should be indented. Not a priority, just noting it. 
It looks like the line below should have been indented instead.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 2504ee72aa0d8b47485279070a2a24bd5c5c00cb
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 9eea7a4fbea9e4aaefb54170e6762c8a3ef33fca
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: c0321ace08dcb0ebc51040657385c3bfad1c1540
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 2504ee72aa0d8b47485279070a2a24bd5c5c00cb
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 9eea7a4fbea9e4aaefb54170e6762c8a3ef33fca
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: c0321ace08dcb0ebc51040657385c3bfad1c1540
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2e947ac762770b20a00f9d0f22f0c8f946b8ca7a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Pull request merged**